ansible/roles/vm_set/library/vm_topology.py

```diff
@@ -497,6 +521,36 @@ def unbind_vm_backplane(self):
 
         return
 
+    def bind_vs_dut_ports(self, br_name, dut_ports):
+        # dut_ports is a list of port on each DUT that has to be bound together. eg. 30,30,30 - will bind ports
+        # 30 of each DUT together into bridge br_name
+        # Also for vm, a dut's ports would be of the format <dut_hostname>-<port_num + 1>. So, port '30' on vm with
+        # name 'vlab-02' would be 'vlab-02-31'
+        br_ports = VMTopology.get_ovs_br_ports(br_name)
+        for dut_index, a_port in enumerate(dut_ports):
+            dut_name = self.duts_name[dut_index]
+            port_name = "{}-{}".format(dut_name, (a_port + 1))
+            br = VMTopology.get_ovs_bridge_by_port(port_name)
+            if br is not None and br != br_name:
+                VMTopology.cmd('ovs-vsctl del-port %s %s' % (br, port_name))
```

Comment:
Do we have legitimate case where a port is being put into more than 1 bridge? This if block is making sure that a port can only be in one bridge.

Reply:
This should never happen, but I copied this from the 'bind_ovs_ports' method. We can remove the check if required.